### ISSUE TITLE
Status Tab should reflect 'blocked exhaust' state

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#1724]
Now that various objects have exhaust values that allow output from fuel could the status tab of the object reflect when the exhaust value is blocked by wall or ceiling.    We are getting numerous requests about why the out will not work and every time it is because the object is placed where the exhaust is against a  wall or ceiling.  Without a note on the status tab the the exhaust is blocked the player is clueless as to why the object is not functioning.

**Comment by MarjieAVolk** [comment]
Duplicate of StrangeLoopGames/EcoIssues#1690
